test/verifiedTests/analysis/testExploration/testFindRxnsFromGenes.m

```diff
@@ -0,0 +1,53 @@
+% The COBRAToolbox: testFindRxnsFromGenes.m
+%
+% Purpose:
+%     - tests that reactions are found when providing a list of genes.
+%
+% Authors:
+%     - Original file: Stefania Magnusdottir August 2017
+%
+
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testFindRxnsFromGenes'));
+cd(fileDir);
+
+% load model
+load('Test_EcoliCoreModel.mat')
```

Comment:
I see. Thanks for changing that. 

May you use `readCbModel` instead of `load` and then `convertOldStyleModel(model);`? Just saw that now.